product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,313 @@
+        ```
+      
+     b. Restart the pemagent service:
+
+        ```shell
+        systemctl restart pemagent
+        ```
+
+
+## Trusted third-party CA certificates and keys
+
+You can replace the PEM self-signed SSL certificates and keys with trusted third-party CA certificates and keys. 
+
+After obtaining the trusted third-party CA certificates and keys: 
+
+- Replace PEM self-signed certificates and keys with trusted CA-signed certificates and keys
+- Disable CRL usage by the server if the trusted third-party CA does not provide CRL
+
+### Replace the server self-signed SSL certificates with the trusted third-party CA certificates
+
+1. Take the backup of old server certificate and key files:
+
+   ```shell
+   # Running as root
+   mkdir /var/lib/edb/as<x>/data/certs
+   cd /var/lib/edb/as<x>/data/
+   mv server.* root.* ca_* /var/lib/edb/as<x>/data/certs
+   ```
+
+1. Generate a private key for the server:
+
+   ```shell
+   openssl genrsa -out server.key 4096 
+   ```
+
+1. Generate a Certificate Signing Request (CSR) for the server:
+
+   ```shell
+   openssl req -new -key server.key -out server.csr -subj '/C=IN/ST=MH/L=Pune/O=EDB/CN=PEM'
+   ```
+   
+   Where `-subj` can be provided as per your requirements. We recommend using the `Hostname/Domain Qualified Full Name` of the PEM server host for `CN`.
+   
+1. Obtain the third-party CA certificate (`trusted_ca.crt`) from a trusted certificate authority (CA). 
+
+1. Ask your CA to sign the CSR and generate the server certificate for you. 
+
+1. Verify the details of the new server certificate aren't tampered with and match your provided details:
+
+   ```shell
+   openssl x509 -noout -text -in server.crt
+   ```
+
+1. Use the new certificate obtained from CA as the `root.crt` file:
+
+   ```shell
+   cp ca_certificate.crt root.crt
+   ```
+
+1. Copy new `root.crt`, `server.key`, and `server.crt` to the data directory of the backend database server:
+
+   ```shell
+   cp root.crt server.key server.crt /var/lib/edb/as<x>/data
+   ```
+
+1. Change the owner and permissions of new certificates and key files same as the data directory:
+
+   ```shell
+   cd /var/lib/edb/as<x>/data/
+   chown enterprisedb server.* root.crt ca_certificate.crt
+   chmod 600 server.* root.crt ca_certificate.crt
+   ```
+
+    
+   !!! Note
+       Do not restart the PEM server now, or else all the registered agents will stop working.
+
+1. Replace each PEM agent SSL certificates with the trusted third-party CA certificates. For more information, see this [section](/pem/latest/certificates/#replace-the-agent-self-signed-ssl-certificates-with-the-trusted-third-party-ca-certificates).
+
+1. Restart the PEM server.
+
+   - On Linux:
+
+   ```shell
+   # Running as root
+   systemctl start edb-as-<x>
+   ```
+
+   Restarting the backend database server, restarts the PEM server.
+
+### Replace the agent self-signed SSL certificates with the trusted third-party CA certificates
```

Comment:
These instructions only work if you have previously followed the instructions above. I think this is reasonable as most people will want to change both, but we should probably note in this section that you need to have completed the instructions above first - specifically adding the new `root.crt`

Reply:
I discussed this with @asheshv so he should be be able to provide the commands